src/renderer/components/FtIconButton/FtIconButton.scss

```
@@ -9,12 +9,16 @@
 
 .iconButton {
+  display: inline-flex;
+  align-items: center;
+  justify-content: center;
   background-color: transparent;
   border-radius: 50%;
   border-style: none;
   color: inherit;
   cursor: pointer;
-  line-height: 1em;
+  // Avoid text-baseline strut so FA / Iconify glyphs share the same geometric center.
```

Stylelint (17.14.1)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)